TESTPLAN.md

```diff
@@ -1,117 +1,40 @@
-# Test Plan for the 57-Perception-Study Addon
+# Test plan for this add-on
 
-## User Experience / Functionality
+## Manual / QA TEST Instructions
+
+### Preparations
 
-During INSTALL ONLY users see:
+* Download a Release version of Firefox (Release is required for the recommendation heuristics to work)
```

Comment:
What do you mean by "recommendation heuristics"? Can you expand on that a little here?

Reply:
Sorry, a leftover from taar-v2, removing...